x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_form.tsx

```diff
@@ -297,6 +302,27 @@ export const AgentPolicyForm: React.FunctionComponent<Props> = ({
           }}
         />
       </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
+        title={<h4>{unenrollmentTimeoutText}</h4>}
+        description={
+          <FormattedMessage
+            id="xpack.fleet.agentPolicyForm.unenrollmentTimeoutDescription"
+            defaultMessage="An optional timeout in seconds. If provided, an agent will automatically unenroll after being gone for this period of time."
```

Comment:
Please let me know if you have any suggestions to improve the language here, @mostlyjason @hbharding. I'm not confident in this phrasing and am happy to update.